src/directive.js

```diff
@@ -204,7 +224,7 @@ let init = function (el, binding, vnode) {
 }
 
 export default {
-  bind: function (el, binding, vnode) {
+  inserted: function (el, binding, vnode) {
```

Comment:
not sure if this is the best option, it is needed to be able to select a parent element for the container, as an alternative, we could separate the bind and inserted hooks, and somehow get the scroller to work